src/Symfony/Component/Form/Tests/Extension/Core/EventListener/MergeCollectionListenerTest.php

```diff
@@ -34,7 +34,7 @@ abstract protected function getBuilder($name = 'name');
 
     protected function getForm($name = 'name', $propertyPath = null)
     {
-        $propertyPath = $propertyPath ?: $name;
+        $propertyPath ??= $name;
```

Comment:
what if the property path variable is the empty string ?

Reply:
This is a test. If the empty string were a problem, the CI should tell us. 🙂

Reply:
This is a test case so we'll figure out when this happens. Test are green :)

Reply:
oh, I missed the fact it is a test. All good then.